scripts/nginx/sabnzbd.sh

```diff
@@ -8,14 +8,25 @@
 #   changes/dates in source files. Any modifications to our software
 #   including (via compiler) GPL-licensed code must also be made available
 #   under the GPL along with build & install instructions.
-MASTER=$(cut -d: -f1 < /root/.master.info)
+user=$(cut -d: -f1 < /root/.master.info)
+active=$(systemctl is-active sabnzbd)
+
+systemctl stop sabnzbd
+
```

Comment:
The two sabnzbd issues are truly nitpicks. If sabnzbd is not active, stopping it won't do anything. If it is active, it will be stopped, which it needs to be to modify the config files.

Similary restart will start the service if it's not currently started, so there is no detriment to using restart in place of start here.